doc/10-icinga-template-library.md

```diff
@@ -3278,6 +3278,146 @@ printer_health_oids		| **Optional.** A list of oids which are downloaded and wri
+thola_api_address                  | **Required.** Address of the Thola API to connect to
+thola_device_address               | **Required.** The host's address. Defaults to "$address$" if the host's address attribute is set, “$address6$” otherwise
+thola_device_snmp_community        | **Optional.** SNMP community of the device
+thola_identify_model               | **Optional.** Model that is compared to the actual model of the device
+thola_identify_os_version          | **Optional.** OS-version that is compared to the actual OS-version of the device
+thola_identify_vendor              | **Optional.** Vendor that is compared to the actual vendor of the device
+thola_identify_serial_number       | **Optional.** Serial number that is compared to the actual serial number of the device
+thola_identify_discover_retries    | **Optional.** The number of discover retries before aborting
+thola_identify_discover_timeouts   | **Optional.** The number of discover timeouts before aborting
+
+> **Note**:
+>
+> One of the variables 'thola_identify_model', 'thola_identify_os_version', 'thola_identify_vendor' or 'thola_identify_serial_number' must be set
+
+##### thola-memory-usage <a id="plugin-contrib-command-thola-memory-usage"></a>
+
+Checks the memory usage of a device.
+
+Custom variables passed as [command parameters](03-monitoring-basics.md#command-passing-parameters):
+
+Name                           | Description
+-------------------------------|-----------------------------------------------------
+thola_api_address              | **Required.** Address of the Thola API to connect to
+thola_device_address           | **Required.** The host's address. Defaults to "$address$" if the host's address attribute is set, “$address6$” otherwise
+thola_device_snmp_community    | **Optional.** SNMP community of the device
+thola_memory_usage_critical    | **Optional.** Critical threshold for the memory usage
+thola_memory_usage_warning     | **Optional.** Warning threshold for the memory usage
+
+##### thola-sbc <a id="plugin-contrib-command-thola-sbc"></a>
+
+Checks special metrics from sbc network devices.
+
+Custom variables passed as [command parameters](03-monitoring-basics.md#command-passing-parameters):
+
+Name                                     | Description
+-----------------------------------------|-----------------------------------------------------------
+thola_api_address                        | **Required.** Address of the Thola API to connect to
+thola_device_address                     | **Required.** The host's address. Defaults to "$address$" if the host's address attribute is set, “$address6$” otherwise
+thola_device_snmp_community              | **Optional.** SNMP community of the device
+thola_sbc_system_health_score_critical   | **Optional.** Critical threshold for the health score
+thola_sbc_system_health_score_warning    | **Optional.** Warning threshold for the health score
```

Comment:
At least here the unit is not very clear (to me).

Reply:
Do you consider the other thresholds' units obvious?

Reply:
Thola currently doesn't support threshold units for current and voltage, but I added the units for CPU load and memory usage

Reply:
Sure, no problem that you can’t choose the unit, e.g. it's always given in volts. But (at least in that case) I should know that I have to specify it in volts, don’t I? (see also e.g. #8759)

Reply:
Yeah, this might be better, I just added it